include/uv.h

```diff
@@ -1372,7 +1372,7 @@ UV_EXTERN uint64_t uv_hrtime(void);
 
 UV_EXTERN void uv_disable_stdio_inheritance(void);
 
-UV_EXTERN int uv_dlopen(const char* filename, uv_lib_t* lib);
+UV_EXTERN int uv_dlopen(const char* filename, uv_lib_t* lib, int glob);
```

Comment:
Can you make this a flags field?  See e.g. `uv_fs_event_start()` for an example.  That way we can add flags later without having to tack on yet more arguments.